modules/serverless-rn-1-17-0.adoc

```diff
@@ -0,0 +1,53 @@
+[id="serverless-rn-1-17-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.17.0
+
+[id="new-features-1-17-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.23.0.
+* {ServerlessProductName} now uses Knative Eventing 0.23.0.
+* {ServerlessProductName} now uses Kourier 0.23.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.23.0.
+* {ServerlessProductName} now uses Knative Kafka 0.23.0.
+* The `kn func` CLI plug-in now uses `func` 0.17.0.
+* In the upcoming {ServerlessProductName} 1.19.0 release, the URL scheme of external routes will default to HTTPS for enhanced security.
++
+If you would like to begin using HTTPS for external routes by default in earlier versions, you can update the `KnativeServing` custom resource definition (CRD) to contain the following YAML:
++
+[source,yaml]
+----
+...
+spec:
+  config:
+    network:
+      defaultExternalScheme: "https"
+...
+----
```

Comment:
This doesn't work in 1.17 yet, it will only work in 1.18.

Reply:
OK, I'll just leave in the first line to say it's coming then?

Reply:
Yes, though we might want to tell people how to do the opposite today (even though it will be a noop until 1.19). We can tell them to set the scheme to `http` explicitly, if they want to stick to it after we change the default.